e2e/resize.go

```diff
@@ -113,7 +113,7 @@ func resizePVCAndValidateSize(pvcPath, appPath string, f *framework.Framework) e
 		return err
 	}
 	// wait for application pod to come up after resize
-	err = waitForPodInRunningState(app.Name, app.Namespace, f.ClientSet, deployTimeout)
+	err = waitForPodInRunningState(app.Name, app.Namespace, f.ClientSet, deployTimeout, "")
```

Comment:
Instead of passing `""` as `expectedError`, define a const `noError = ""` and pass that. It will be much easier to understand when using the function in additional tests